Library/Homebrew/cask/test/test_helper.rb

```diff
@@ -50,7 +50,7 @@ class TestCask < Cask; end
 end
 
 # pretend that the caskroom/cask Tap is installed
-FileUtils.ln_s HOMEBREW_LIBRARY.join("Taps", "caskroom", "homebrew-cask"), Tap.fetch("caskroom", "cask").path
+FileUtils.ln_s Pathname.new(ENV["HOMEBREW_LIBRARY"]).join("Taps", "caskroom", "homebrew-cask"), Tap.fetch("caskroom", "cask").path
```

Comment:
Well, we need to link the *actual* `caskroom/cask` tap, and since `HOMEBREW_LIBRARY` is pointing to the one in the test environment, we need to use the environment variable used by `brew cask-tests`. Sooner or later, once we have converted the tests to only use the `test` tap, we can get rid of this completely.